.github/workflows/docker-workflow.yml

```diff
@@ -2,6 +2,14 @@ name: Build, Deploy and release Docker images
 on:
   workflow_call:
     inputs:
+      docker_registry:
+        required: false
+        type: string
+        default: registry.aws.cloud.mov.ai
+      public_registry:
```

Comment:
Maybe we dont need to have 2 inputs for the registry. Simply that boolean you added should be enough to differentiate.